images/sriov-network-operator.yml

```diff
@@ -0,0 +1,20 @@
+config: wip
+container_yaml:
+  go:
+    modules:
+    - module: github.com/openshift/sriov-network-operator
+content:
+  source:
+    dockerfile: Dockerfile.rhel7
```

Comment:
this file should exist in root dir of https://github.com/openshift/sriov-network-operator

Reply:
I believe ticket is wrong since I didnt find .rhel7 file in repo, but you can always try

Reply:
Yes, I thought about it, but got confused by what was filled in the ticket:
![Screenshot from 2019-06-13 11-21-34](https://user-images.githubusercontent.com/190616/59420616-79a24300-8dcd-11e9-81d6-8226cbc853fe.png)
Changed to use only `Dockerfile` now.